Cython/Utils.py

```diff
@@ -579,6 +579,9 @@ def add(self, e):
             self._list.append(e)
             self._set.add(e)
 
+    def __bool__(self):
+        return bool(self._set)
+
 
```

Comment:
We still need `__nonzero__` in Py2.
```suggestion
    def __bool__(self):
        return bool(self._set)
        
    __nonzero__ = __bool__

```